include/grpcpp/health_check_service_interface.h

```diff
@@ -19,39 +19,22 @@
 #ifndef GRPCPP_HEALTH_CHECK_SERVICE_INTERFACE_H
 #define GRPCPP_HEALTH_CHECK_SERVICE_INTERFACE_H
 
-#include <grpcpp/support/config.h>
+#include <grpcpp/health_check_service_interface_impl.h>
 
 namespace grpc {
 
 const char kHealthCheckServiceInterfaceArg[] =
```

Comment:
I'm... surprised this actually works. This is unrelated to this PR, but that's really not supposed to work like this.

The proper way would be to define `kHealthCheckServiceInterfaceArg` in the header like so:

```
extern const char kHealthCheckServiceInterfaceArg[];
```

And then, in a .cc file, have the actual declaration. The way this is currently done means the symbol might get duplicated at link time.

cc @vjpai.